scripts/release.ts

```diff
@@ -167,10 +167,25 @@ function readPackageJSON(packageDir) {
   );
 }
 
-function diffJson(from: string = "latest", to?: string): string {
+function toPrettyJson(sourceFp: string, destFp: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    const jqProcess = spawn("jq", [".", sourceFp]);
```

Comment:
any reason not to use `node-jq`?

Reply:
ah maybe streams?

Reply:
Actually, this whole thing was silly: the original reason for using `jq` is no longer needed. I can just read, parse, and stringify to get the same result. 🤦